auth/src/main/java/org/aerogear/auth/utils/TokenPersistor.java

```diff
@@ -0,0 +1,31 @@
+package org.aerogear.auth.utils;
+
+import org.aerogear.auth.credentials.TokenCredentials;
+
+/**
+ * Saves, retrieves and delete a token.
+ */
+public class TokenPersistor {
+    private TokenPersistor() {
+    }
+
+    public static TokenCredentials load() {
+        throw new IllegalStateException("Not yet implemented");
+    }
+
+    /**
+     * Saves a token
+     * @param token token to be saved
+     */
+    public static void save(TokenCredentials token) {
```

Comment:
@wei-lee Would it be a good idea to pass here the Principal? (it already has a method to extract the token credential)

Reply:
I think pass in token is ok for now. The class is called "TokenPersistor".